gpflow/kernels/base.py

```diff
@@ -26,7 +26,7 @@
 import tensorflow as tf
 
 
-class Kernel(tf.Module):
+class Kernel(tf.Module, metaclass=abc.ABCMeta):
```

Comment:
would it be better to do
```suggestion
class Kernel(tf.Module, abc.ABC):
```
instead ?